src/utils/telephoneLink.jsx

```diff
@@ -0,0 +1,51 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import A from '../atoms/a';
+
+const propTypes = {
+    className: PropTypes.string,
+    formattedNumber: PropTypes.string,
+    number: PropTypes.string,
+    tabIndex: PropTypes.number,
+};
+
+const defaultProps = {
+    className: undefined,
+    number: undefined,
+    formattedNumber: undefined,
+    tabIndex: -1,
+};
+
+function TelephoneLink(props) {
+    const {
+        className,
+        formattedNumber,
+        number,
+        tabIndex,
+    } = props;
+
+    if (!number) {
+        return null;
+    }
+
+    const onClick = (event) => {
+        event.stopPropagation();
+
+        window.location.href = `tel:${number}`;
+    };
```

Comment:
Any reason to do this, versus actually use a plain-old regular `<a> with this `href`?

Reply:
Right now the A component is not an actual `<a>`.

Reply:
Switched too `<a>`, added `href` attribute, and remove the `onClick` method.

Reply:
Fixed as requested in ba0497c.  Resolving.